2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_221113-28bq3u5x/logs/debug.log
2022-03-25 22:11:13,744 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_221113-28bq3u5x/logs/debug-internal.log
2022-03-25 22:11:13,745 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-25 22:11:13,745 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 22:11:13,745 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 28asnehw
2022-03-25 22:11:13,747 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_peak_freq_project/28asnehw
2022-03-25 22:11:13,748 INFO    MainThread:12299 [jupyter.py:save_history():429] not saving jupyter history
2022-03-25 22:11:13,748 INFO    MainThread:12299 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-25 22:11:13,748 INFO    MainThread:12299 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-25 22:11:13,748 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 22:11:13,763 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-25 22:11:14,090 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:11:14,393 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:11:14,779 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:11:14,886 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 10435
}

2022-03-25 22:11:15,022 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 10435
}

2022-03-25 22:11:15,127 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4539
  total_bytes: 10435
}

2022-03-25 22:11:15,245 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4539
  total_bytes: 10435
}

2022-03-25 22:11:15,356 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:15,476 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:15,586 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:15,695 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:15,816 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:16,581 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}

2022-03-25 22:11:17,059 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10435
  total_bytes: 10435
}
local_info {
}

2022-03-25 22:11:18,219 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 22:11:18,219 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 22:11:18,221 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-25 22:11:18,223 INFO    MainThread:12299 [wandb_init.py:init():492] starting backend
2022-03-25 22:11:18,223 INFO    MainThread:12299 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-25 22:11:18,226 INFO    MainThread:12299 [backend.py:ensure_launched():219] starting backend process...
2022-03-25 22:11:18,244 INFO    MainThread:12299 [backend.py:ensure_launched():224] started backend process with pid: 12407
2022-03-25 22:11:18,246 INFO    MainThread:12299 [wandb_init.py:init():501] backend started and connected
2022-03-25 22:11:18,266 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-25 22:11:18,266 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-25 22:11:18,266 INFO    MainThread:12299 [wandb_init.py:init():565] updated telemetry
2022-03-25 22:11:18,300 INFO    MainThread:12299 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-25 22:11:19,836 INFO    MainThread:12299 [wandb_run.py:_on_init():1759] communicating current version
2022-03-25 22:11:19,943 INFO    MainThread:12299 [wandb_run.py:_on_init():1763] got version response 
2022-03-25 22:11:19,943 INFO    MainThread:12299 [wandb_init.py:init():625] starting run threads in backend
2022-03-25 22:11:22,059 INFO    MainThread:12299 [wandb_run.py:_console_start():1733] atexit reg
2022-03-25 22:11:22,060 INFO    MainThread:12299 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-25 22:11:22,060 INFO    MainThread:12299 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-25 22:11:22,060 INFO    MainThread:12299 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-25 22:11:22,061 INFO    MainThread:12299 [wandb_init.py:init():664] run started, returning control to user process
2022-03-25 22:11:22,061 INFO    MainThread:12299 [wandb_watch.py:watch():47] Watching
2022-03-25 22:14:47,918 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set batch_size = 40 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b1265a30>>
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb batch_size 40 None
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set epochs = 45 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b1265a30>>
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb epochs 45 None
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set lr = 0.00483244283125746 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b1265a30>>
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb lr 0.00483244283125746 None
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b1265a30>>
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b1265a30>>
2022-03-25 22:14:47,919 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_221520-qehq02fs/logs/debug.log
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_221520-qehq02fs/logs/debug-internal.log
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f83b676f1c0>
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 22:15:20,221 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 28bq3u5x
2022-03-25 22:15:20,224 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/28bq3u5x
2022-03-25 22:15:20,224 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 22:15:20,225 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-25 22:15:20,377 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:15:20,498 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:15:20,748 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-25 22:15:21,016 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 157148
}

2022-03-25 22:15:21,124 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 163044
}

2022-03-25 22:15:21,237 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 163044
}

2022-03-25 22:15:21,342 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 163044
}

2022-03-25 22:15:21,453 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12913
  total_bytes: 163044
}

2022-03-25 22:15:21,568 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 160369
  total_bytes: 163044
}

2022-03-25 22:15:21,681 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:21,793 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:21,898 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:22,006 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:22,119 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:22,229 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:23,108 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}

2022-03-25 22:15:23,585 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 163044
  total_bytes: 163044
}
local_info {
}

2022-03-25 22:15:24,775 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 22:15:24,778 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 22:15:24,781 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
